api/v1alpha1/osbuildenvconfig_types.go

```diff
@@ -40,6 +40,9 @@ type OSBuildEnvConfigSpec struct {
 	// S3Service holds the configuration needed to connect to the S3 service
 	// +kubebuilder:validation:Required
 	S3Service S3ServiceConfig `json:"s3Service"`
+	// ContainerRegistryService holds the configuration needed to upload container images to the registry
+	// +kubebuilder:validation:Required
+	ContainerRegistryService ContainerRegistryServiceConfig `json:"containerRegistryService"`
```

Comment:
Are you sure it's a mandatory field? what about qcow2 or other image types?

Reply:
Not providing a Container Registry means the system will not be able to produce edge-container or edge-installer. We can make it optional, but then the OSBuildConfig controller will need to check and fail such requests. Do we want to do that?

Reply:
Mmm not sure what is better
let's keep it like that